File: frontend/src/models/Set.js
export default class Set {

    constructor(weight = 0.0, reps = 0, logged = false) {
        this.weight = weight;
        this.reps = reps;
        this.logged = false;
    }

    getWeight() {
        return this.weight;
    }

    setWeight(weight) {
        this.weight = weight;
    }

    getReps() {
        return this.reps;
    }

    setReps(reps) {
        this.reps = reps;
    }

    getLogged() {
        return this.logged;
    }

    setLogged(logged) {
        this.logged = logged;
    }
}

File: backend/src/main/java/com/j_mikolajczyk/backend/models/Set.java
package com.j_mikolajczyk.backend.models;

public class Set {
    private double weight;
    private int reps;
    private boolean logged;

    public Set() {
        this.weight = 0.0;
        this.reps = 0;
        this.logged = false;
    }

    public Set(double weight, int reps) {
        this.weight = weight;
        this.reps = reps;
        this.logged = false;
    }

    public double getWeight() {
        return this.weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    public int getReps() {
        return this.reps;
    }

    public void setReps(int reps) {
        this.reps = reps;
    }

    public boolean getLogged() {
        return this.logged;
    }

    public void setLogged(boolean logged) {
        this.logged = logged;
    }
    
}
